nucypher/blockchain/eth/token.py

```diff
@@ -271,12 +281,7 @@ def validate_value(self, raise_on_fail: bool = True) -> Union[bool, Tuple[Tuple[
         rulebook = (
 
             (self.minimum_nu <= self.value,
-             'Stake amount too low; ({amount}) must be at least {minimum}'
-             .format(minimum=self.minimum_nu, amount=self.value)),
-
-            (self.maximum_nu >= self.value,
-             'Stake amount too high; ({amount}) must be no more than {maximum}.'
-             .format(maximum=self.maximum_nu, amount=self.value)),
+             f'Stake amount too low; ({self.minimum_nu}) must be at least {self.value}'),
```

Comment:
Seems like the order should be switched
```suggestion
             f'Stake amount too low; ({self.value}) must be at least {self.minimum_nu}'),
```

Reply:
Hahah - well that's confusing.

Reply:
:heavy_check_mark: